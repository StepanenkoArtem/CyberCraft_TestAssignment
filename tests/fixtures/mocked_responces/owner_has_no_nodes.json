{
    "user": {
        "login": "owner_has_no_nodes",
        "name": "Empty Owner"
    },
    "repositoryOwner": {
        "repositories": {
            "nodes": []
        }
    }
}
