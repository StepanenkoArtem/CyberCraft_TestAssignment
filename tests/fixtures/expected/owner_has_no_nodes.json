{
    "login": "owner_has_no_nodes",
    "name": "Empty Owner",
    "nodes": []
}
